### PR TITLE
feat/fix(apcd): support isolated local CMS database and volumes

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Provide instruction and file to support an isolated APCD database (rather than sharing `core_cms_postgres`).

<details>

If developer already has a [TACC/Core-Portal](https://github.com/TACC/Core-Portal), and uses `docker-compose.dev.yml`, then they may experience a conflict error for `core_cms_postgres` container/host/volume name.

If developers work on multiple [TACC/Core-CMS](https://github.com/TACC/Core-CMS)-based projects, and they build more than one, they may encounter a conflict error for `core_cms_postgres` container/host/volume name.

If developer works on multiple [TACC/Core-CMS](https://github.com/TACC/Core-CMS)-based projects, and they clone remote database to local database (for accurate testing), they would overwrite the databases other local instances of CMS projects are using.

</details>

## Related

- affects **current** developers locally testing APCD *and* a [TACC/Core-CMS](https://github.com/TACC/Core-CMS)-based project
- affects **new** developers locally testing APCD *and* [TACC/Core-Portal](https://github.com/TACC/Core-Portal)
- affects **future** developers locally testing APCD *and* a future CMS project in this repo

## Changes

- add alternative docker-compose file
- document usage of that file

## Testing

1. Have a local [TACC/Core-Portal](https://github.com/TACC/Core-Portal) instance.
2. Run a local `apcd-cms` instance __using the original docker compose dev file__.
3. ⓧ Verify conflict error.
2. Run a local `apcd-cms` instance __using the new alternative docker compose dev file__.
4. ✓ Verify __no__ conflict error.

## UI

N/A

## Notes

Alternatively, I could have a sample such file in root, and document how to clone and edit it per project as needed.

### CODE REVIEW

File: apcd-cms/docker-compose.dev.unique.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# To isolate APCD CMS development into its own database and volumes
```
Comment (rstijerina) [inline apcd-cms/docker-compose.dev.unique.yml:1]: This is for local development, right? Does the same need to be added to Core-Portal-Deployments?

Comment (wesleyboar) [reply apcd-cms/docker-compose.dev.unique.yml:1]: Yes, it is for local development (for `/apcd-cms`﹡). No, I do not see it needed for Core-Portal-Deployments.

<sup>﹡ Also, I'd rather **not** duplicate this _per project_ (no others here yet), but I have **not** yet found a _successful_ way to put the bulk of this in root.</sup>

Comment (wesleyboar) [reply apcd-cms/docker-compose.dev.unique.yml:1]: Do **you** see a use for it in Core-Portal-Deployments?

I see a use if more than one CMS were on a server, but that doesn't sound like something we do.

Comment (wesleyboar) [reply apcd-cms/docker-compose.dev.unique.yml:1]:
```suggestion
# To isolate local APCD CMS development into its own database and volumes
```

**Comment by wesleyboar** [comment]
Declined, because by the time I return to this, there will be a different solution.

https://github.com/TACC/Core-CMS/pull/577 already may provide something to learn from.

Also, TACC/Core-CMS-Custom is not a certain future for the CMS projects. It needs further peer review.

Related
- https://github.com/TACC/Core-CMS-Custom/pull/36

**Comment by wesleyboar** [comment]
This can now be done by following how #161 did it for `matcssi_cms`.

**Pull request closed without merging**